workflows/amplicon/qiime2/qiime2-I-import/QIIME2-Ia-multiplexed-data-single-end.ga

```diff
@@ -189,7 +189,7 @@
                 },
                 "RenameDatasetActionimported_data": {
                     "action_arguments": {
-                        "newname": "input-data-single-end"
+                        "newname": "input-data-single read"
```

Comment:
The idea was not to deploy the workflows and wait on the next iteration. Otherwise all of them need bumping. In this case though we should probably go with
```suggestion
                        "newname": "input-data-single-read"
```